wandb/sdk/service/streams.py

```diff
@@ -248,7 +248,7 @@ def _process_drop(self, action: StreamAction) -> None:
     def _on_probe_exit(self, probe_handle: MailboxProbe, stream: StreamRecord) -> None:
         handle = probe_handle.get_mailbox_handle()
         if handle:
-            result = handle.wait(timeout=0, release=False)
+            result = handle.wait(timeout=0)
```

Comment:
hmm maybe this one is fine to keep?
@raubitsj

Reply:
good catch, let me open up a separate PR with this fix